### ISSUE TITLE
Bwc test failures caused by DeprecationIndexingTemplateRegistry ?

### ISSUE THREAD

**Issue by javanna** [elastic/elasticsearch#70020]
There's quite some bwc test failures today with this error:

```
Execution failed for task ':qa:rolling-upgrade:v7.3.2#upgradedClusterTest'.
> process was found dead while waiting for ports files, node{:qa:rolling-upgrade:v7.3.2-2}
```

**Build scan**:

https://gradle-enterprise.elastic.co/s/sehit3eudmhna

**Applicable branches**:

7.x

I see this error in the logs:

```

» [2021-03-05T08:51:01,096][ERROR][o.e.x.c.t.IndexTemplateRegistry] [v7.3.2-0] error adding index template [.deprecation-indexing-mappings] from [/org/elasticsearch/xpack/deprecation/deprecation-indexing-mappings.json] for [deprecation] |  
-- | --
  | »  org.elasticsearch.transport.RemoteTransportException: [v7.3.2-1][127.0.0.1:41501][cluster:admin/component_template/put] |  
  | »  Caused by: org.elasticsearch.transport.ActionNotFoundTransportException: No handler for action [cluster:admin/component_template/put] |  
  | »  	at org.elasticsearch.transport.InboundHandler.handleRequest(InboundHandler.java:169) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.transport.InboundHandler.messageReceived(InboundHandler.java:121) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.transport.InboundHandler.inboundMessage(InboundHandler.java:105) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.transport.TcpTransport.inboundMessage(TcpTransport.java:660) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.transport.netty4.Netty4MessageChannelHandler.channelRead(Netty4MessageChannelHandler.java:62) ~[transport-netty4-client-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:323) ~[netty-codec-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:297) ~[netty-codec-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.handler.logging.LoggingHandler.channelRead(LoggingHandler.java:241) ~[netty-handler-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1408) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:930) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:682) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:582) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:536) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) ~[?:?] |  
  | »  	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at java.lang.Thread.run(Thread.java:835) [?:1.8.0_281]

```

and

```

»  Caused by: org.elasticsearch.transport.ActionNotFoundTransportException: No handler for action [indices:admin/data_stream/delete] |  
-- | --
  | »  	at org.elasticsearch.transport.InboundHandler.handleRequest(InboundHandler.java:169) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.transport.InboundHandler.messageReceived(InboundHandler.java:121) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.transport.InboundHandler.inboundMessage(InboundHandler.java:105) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.transport.TcpTransport.inboundMessage(TcpTransport.java:660) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at org.elasticsearch.transport.netty4.Netty4MessageChannelHandler.channelRead(Netty4MessageChannelHandler.java:62) ~[transport-netty4-client-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:323) ~[netty-codec-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:297) ~[netty-codec-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.handler.logging.LoggingHandler.channelRead(LoggingHandler.java:241) ~[netty-handler-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:352) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1408) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:374) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:360) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:930) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:682) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:582) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:536) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496) ~[netty-transport-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:906) ~[?:?] |  
  | »  	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.49.Final.jar:4.1.49.Final] |  
  | »  	at java.lang.Thread.run(Thread.java:835) [?:1.8.0_281]

```

This seems to be linked with https://github.com/elastic/elasticsearch/issues/69918#issuecomment-791303812

**Comment by mark-vieira** [comment]
I'd keep scrolling, I think the errors further down are the ones causing the cluster from coming up:

```
» [2021-03-05T17:53:24,848][WARN ][o.e.c.c.PublicationTransportHandler] [v7.3.1-2] unexpected error while deserializing an incoming cluster state
»  java.lang.IllegalArgumentException: Unknown NamedWriteable [org.elasticsearch.cluster.NamedDiff][component_template]
»  	at org.elasticsearch.common.io.stream.NamedWriteableRegistry.getReader(NamedWriteableRegistry.java:112) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.common.io.stream.NamedWriteableAwareStreamInput.readNamedWriteable(NamedWriteableAwareStreamInput.java:45) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.NamedDiffableValueSerializer.readDiff(NamedDiffableValueSerializer.java:56) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.NamedDiffableValueSerializer.readDiff(NamedDiffableValueSerializer.java:30) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.DiffableUtils$MapDiff.<init>(DiffableUtils.java:406) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.DiffableUtils$ImmutableOpenMapDiff.<init>(DiffableUtils.java:234) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.DiffableUtils.readImmutableOpenMapDiff(DiffableUtils.java:127) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.metadata.MetaData$MetaDataDiff.<init>(MetaData.java:873) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.metadata.MetaData.readDiffFrom(MetaData.java:812) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.ClusterState$ClusterStateDiff.<init>(ClusterState.java:883) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.ClusterState.readDiffFrom(ClusterState.java:793) ~[elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.coordination.PublicationTransportHandler.handleIncomingPublishRequest(PublicationTransportHandler.java:420) [elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.cluster.coordination.PublicationTransportHandler.lambda$new$0(PublicationTransportHandler.java:100) [elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:63) [elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.transport.InboundHandler$RequestHandler.doRun(InboundHandler.java:267) [elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:758) [elasticsearch-7.3.1.jar:7.3.1]
»  	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-7.3.1.jar:7.3.1]
»  	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
»  	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
»  	at java.lang.Thread.run(Thread.java:835) [?:?]
```

and

```
» [2021-03-05T17:55:14,210][ERROR][o.e.b.Bootstrap          ] [v7.3.1-2] Exception
»  org.elasticsearch.ElasticsearchException: java.io.IOException: failed to read /dev/shm/elastic+elasticsearch+7.x+bwc/BWC_VERSION/7.3.1/nodes/centos-7&&immutable/qa/rolling-upgrade/build/testclusters/v7.3.1-2/data/nodes/0/_state/global-216.st
»  	at org.elasticsearch.ExceptionsHelper.maybeThrowRuntimeAndSuppress(ExceptionsHelper.java:158) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.gateway.MetadataStateFormat.loadGeneration(MetadataStateFormat.java:403) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.gateway.MetaStateService.loadFullState(MetaStateService.java:71) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.gateway.GatewayMetaState.start(GatewayMetaState.java:136) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.node.Node.start(Node.java:836) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.bootstrap.Bootstrap.start(Bootstrap.java:324) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:419) [elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:159) [elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:150) [elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:75) [elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:116) [elasticsearch-cli-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cli.Command.main(Command.java:79) [elasticsearch-cli-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:115) [elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:81) [elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  Caused by: java.io.IOException: failed to read /dev/shm/elastic+elasticsearch+7.x+bwc/BWC_VERSION/7.3.1/nodes/centos-7&&immutable/qa/rolling-upgrade/build/testclusters/v7.3.1-2/data/nodes/0/_state/global-216.st
»  	at org.elasticsearch.gateway.MetadataStateFormat.loadGeneration(MetadataStateFormat.java:397) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	... 12 more
»  Caused by: org.elasticsearch.common.xcontent.XContentParseException: [-1:26235] [index_lifecycle] failed to parse field [policies]
»  	at org.elasticsearch.common.xcontent.ObjectParser.parseValue(ObjectParser.java:520) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.ObjectParser.parseSub(ObjectParser.java:530) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.ObjectParser.parse(ObjectParser.java:313) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.ConstructingObjectParser.parse(ConstructingObjectParser.java:160) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.xpack.ilm.IndexLifecycle.lambda$getNamedXContent$4(IndexLifecycle.java:236) ~[?:?]
»  	at org.elasticsearch.common.xcontent.NamedXContentRegistry$Entry.lambda$new$0(NamedXContentRegistry.java:52) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.NamedXContentRegistry.parseNamedObject(NamedXContentRegistry.java:129) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.support.AbstractXContentParser.namedObject(AbstractXContentParser.java:398) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cluster.metadata.Metadata$Builder.fromXContent(Metadata.java:1736) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cluster.metadata.Metadata$1.fromXContent(Metadata.java:1781) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cluster.metadata.Metadata$1.fromXContent(Metadata.java:1772) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.gateway.MetadataStateFormat.read(MetadataStateFormat.java:291) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.gateway.MetadataStateFormat.loadGeneration(MetadataStateFormat.java:393) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	... 12 more
»  Caused by: org.elasticsearch.common.xcontent.XContentParseException: [-1:26235] [policies] failed to parse field [ml-size-based-ilm-policy]
»  	at org.elasticsearch.common.xcontent.ObjectParser.lambda$declareNamedObjects$13(ObjectParser.java:432) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.ObjectParser.lambda$declareNamedObjects$14(ObjectParser.java:444) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.ObjectParser.parseValue(ObjectParser.java:518) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.ObjectParser.parseSub(ObjectParser.java:530) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.ObjectParser.parse(ObjectParser.java:313) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.ConstructingObjectParser.parse(ConstructingObjectParser.java:160) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.xpack.ilm.IndexLifecycle.lambda$getNamedXContent$4(IndexLifecycle.java:236) ~[?:?]
»  	at org.elasticsearch.common.xcontent.NamedXContentRegistry$Entry.lambda$new$0(NamedXContentRegistry.java:52) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.NamedXContentRegistry.parseNamedObject(NamedXContentRegistry.java:129) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.common.xcontent.support.AbstractXContentParser.namedObject(AbstractXContentParser.java:398) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cluster.metadata.Metadata$Builder.fromXContent(Metadata.java:1736) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cluster.metadata.Metadata$1.fromXContent(Metadata.java:1781) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.cluster.metadata.Metadata$1.fromXContent(Metadata.java:1772) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.gateway.MetadataStateFormat.read(MetadataStateFormat.java:291) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	at org.elasticsearch.gateway.MetadataStateFormat.loadGeneration(MetadataStateFormat.java:393) ~[elasticsearch-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
»  	... 12 more
```

```
»  Caused by: java.lang.IllegalArgumentException: At least one rollover condition must be set.
»  	at org.elasticsearch.xpack.core.ilm.RolloverAction.<init>(RolloverAction.java:73) ~[?:?]
»  	at org.elasticsearch.xpack.core.ilm.RolloverAction.lambda$static$0(RolloverAction.java:46) ~[?:?]
»  	at org.elasticsearch.common.xcontent.ConstructingObjectParser.lambda$new$2(ConstructingObjectParser.java:123) ~[elasticsearch-x-content-7.13.0-SNAPSHOT.jar:7.13.0-SNAPSHOT]
```

**Comment by valeriy42** [comment]
Here seems to be another one https://gradle-enterprise.elastic.co/s/4rjvhdvmw7kps

**Comment by martijnvg** [comment]
The 7.x, 7.12 and 7.11 bwc builds are green now. The deprecation template issue has been resolved via #70057. I will close this issue. Please open a new issue if any of the mentioned problems here re-occur.
